source/features/raw-file-link.tsx

```diff
@@ -10,17 +10,16 @@ const createRawUrl = (href: string): string => {
 };
 
 function addRawButtons(): void {
-	for (const anchor of select.all<HTMLAnchorElement>('.file-header .file-actions > div.BtnGroup [href*=\'/blob/\']:last-child')) {
+	for (const anchor of select.all<HTMLAnchorElement>('.js-file-header-dropdown [data-ga-click^="View file"]')) {
 		anchor.after(
-			<a href={createRawUrl(anchor.pathname)} className="btn btn-sm BtnGroup-item">
-				Raw
+			<a href={createRawUrl(anchor.pathname)} className="pl-5 dropdown-item btn-link" role="menuitem">
+				View raw
 			</a>
 		);
 	}
 }
 
 function init(): void {
-	addRawButtons();
```

Comment:
This needs to be there, or else it will only run if there are many files in a PR